### PR TITLE
Fix ts server error

### PR DESCRIPTION
**Pull request by anpythomas**

### CODE REVIEW

**Comment by ghost** [comment]
When we switched to typescript, it broke production because node cannot directly interpret typescript. To fix this I created a ticket to create a script that will compile our typescript code into javascript. Additionally, with typescript we should use ES module imports and exports.

**Comment by theramirobeltran** [comment]
So this ticket unblocks us, but @dkbkh how close is your change?

**Comment by ghost** [comment]
Just made the pr #8 @theramirobeltran

File: src/server.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import express from 'express';
+const express = require('express')
```
Comment (theramirobeltran) [inline src/server.ts:1]: We can use ESM syntax if just set the 'type: module' in package.json

**Comment by anpythomas** [comment]
Issue resolved, closing out request

**Pull request closed without merging**